soccer/gameplay/plays/testing/test_one_touch_pass.py

```diff
@@ -0,0 +1,34 @@
+import play
+import behavior
+import tactics.one_touch_pass
+import tactics.behavior_sequence
+import robocup
+import constants
+import main
+
+
+## Continually runs a one_touch_pass pass tactic
+class TestOneTouchPass(play.Play):
+
+    ReceiveXCoord = 1
```

Comment:
Instead of defining a receive x and y, you can just define a receive point here.